src/cli/domain/init-template/success.js

```diff
@@ -0,0 +1,20 @@
+module.exports = (logger, componentName, componentPath) => {
```

Comment:
Cause multiline is space sensitive, it will look a bit ugly on that file. I've commited it so that you can see. I think is ok for now as we need to rethink that part anyway.